projects/addon-doc/components/example/example.template.html

```diff
@@ -32,7 +32,11 @@
     </ng-container>
 </h3>
 
-<div class="t-example">
+<div
+    class="t-example"
+    [tuiFullscreen]="fullscreen()"
+    (fullscreenEnabled)="fullscreen.set($event)"
```

Comment:
Let's make it a two-way binding if it can be changed internally

Reply:
fixed